docs/tutorial/part-two/index.md

```diff
@@ -103,67 +103,67 @@ Your project's file structure should now look like this:
 ├── gatsby-browser.js
 ```
 
-> 💡 What is `gatsby-browser.js`? Don't worry about this too much and for now, just know that `gatsby-browser.js` is one of a handful of special files that Gatsby looks for and uses (if they exist). Here, the naming of the file **is** important. If you do want to explore more now, check out [the docs](/docs/browser-apis/).
+> 💡 Apa itu `gatsby-browser.js`? Jangan terlalu khawatir tentang ini untuk sekarang, pahami saja bahwa `gatsby-browser.js` merupakan salah satu dari beberapa berkas khusus yang Gatsby cari dan gunakan (jika ada). Disini, penamaan berkas **merupakan** sesuatu yang penting. Jika Anda ingin mendalami lebih jauh sekarang, Periksa [dokumentasi](/docs/browser-apis/).
 
-2. Import your recently-created stylesheet in the `gatsby-browser.js` file:
+2. Impor stylesheet yang baru saja Anda buat ke dalam berkas `gatsby-browser.js`:
 
 ```javascript:title=gatsby-browser.js
-import "./src/styles/global.css"
+import './src/styles/global.css';
```

Comment:
For code, please dont change it. Revert it to code from Gatsby please

Reply:
I'm sorry...this is caused by my vscode internal linter